### ISSUE TITLE
Vulnerable dependency

### ISSUE THREAD

**Issue by limonte** [BrowserSync/browser-sync#1681]
```sh
cd my-app
npm i browser-sync
npm audit
```

```
                       === npm audit security report ===                        
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ braces                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.3.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ browser-sync                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ browser-sync > micromatch > braces                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/786                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
found 1 low severity vulnerability in 2600 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by dennisreimann** [comment]
This is already addressed in #1659 and #1671.

**Comment by rbiggs** [comment]
You say you've fixed the vulnerabilities, but your NPM version hasn't been updated so when I install 2.26.3 I get the vulnerabilities. You need to bump the version on NPM to give us the fixes.

**Comment by dennisreimann** [comment]
The mentioned pull requests haven't been merged and released yet.

**Comment by shakyShane** [comment]
fixed in 2.26.4 :)
